app/views/LocationTracking.js

```diff
@@ -506,7 +506,9 @@ const styles = StyleSheet.create({
     top: '-13%',
     left: 0,
     right: 0,
+    flex: 1,
```

Comment:
This looks like an unrelated change, what does it do? Perhaps you can revert this change if it's not related.